homeassistant/components/swiss_public_transport/config_flow.py

```diff
@@ -32,28 +52,43 @@
             ),
         ),
         vol.Required(CONF_DESTINATION): cv.string,
+        vol.Optional(CONF_TIME_MODE, default=DEFAULT_TIME_MODE): SelectSelector(
+            SelectSelectorConfig(
+                options=TIME_MODE_OPTIONS,
+                mode=SelectSelectorMode.DROPDOWN,
+                translation_key="time_mode",
+            ),
+        ),
+        vol.Optional(CONF_IS_ARRIVAL, default=DEFAULT_IS_ARRIVAL): SelectSelector(
+            SelectSelectorConfig(
+                options=IS_ARRIVAL_OPTIONS,
+                mode=SelectSelectorMode.DROPDOWN,
+                translation_key="is_arrival",
+            ),
+        ),
```

Comment:
I find the constant names very confusing when it's not a `bool`.
Maybe `CONF_TIME_STATION` and `DEFAULT_TIME_STATION` instead? (or perhaps you have another proposal that sounds more like what it is)

Reply:
True that :)